resident/resident-service/src/main/java/io/mosip/resident/util/ResidentServiceRestClient.java

```diff
@@ -293,11 +279,12 @@ public Object getApi(ApiName apiName, List<String> pathsegments, List<String> qu
 		return result;
 	}
 
+	@Deprecated
 	public RestTemplate getResidentRestTemplate() throws KeyManagementException, NoSuchAlgorithmException, KeyStoreException {
 		logger.info(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.APPLICATIONID.toString(),
 				LoggerFileConstant.APPLICATIONID.toString(), Arrays.asList(environment.getActiveProfiles()).toString());
 
-		TrustStrategy acceptingTrustStrategy = (X509Certificate[] chain, String authType) -> true;
+		/*TrustStrategy acceptingTrustStrategy = (X509Certificate[] chain, String authType) -> true;

```

Comment:
Why are we commenting this code, if the method is mared as deprecated, if no one is using this method, you can simple remove this method.

Reply:
ok. i will remove that.